WalletWasabi.Tests/UnitTests/BranchAndBoundSelectionTests.cs

```diff
@@ -0,0 +1,105 @@
+using NBitcoin;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using WalletWasabi.Blockchain.TransactionOutputs;
+using WalletWasabi.BranchNBound;
+using Xunit;
+
+namespace WalletWasabi.Tests.UnitTests.BranchAndBoundTests
+{
+	public class BranchAndBoundSelectionTests
+	{
+		private static Random Random = new Random();
+		private static List<Money> AvailableCoins = GenList();
```

Comment:
Not sure why this needs to be `static` or a field at all.

Reply:
I want these to be static because this way when we call the random tests, they will run with the same pool of coins.
 This way we work with the pool and we can observe which runs faster with the same data.